Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TokenizingTextBox/TokenizingTextBoxXaml.bind

```diff
@@ -39,7 +39,8 @@
           MaxHeight="104"
           HorizontalAlignment="Stretch"
           TextMemberPath="Text"
-          TokenDelimiter=",">
+          TokenDelimiter=","
+          MaxTokens="3">
```

Comment:
Modifying this MaxTokens value (increasing or decreasing) in a running Sample App is emptying out the TTB of all tokens- does not seem like expected behavior.

Reply:
Wellll, that's not actually because of my code. Whenever you change the content of the XAML tab, the entire sample view is re-rendered. Therefore, we can't really observe the effect of the change event for `MaxTokens` by using the sample app.

Reply:
Yup, @shweaver-MSFT's right. To test dynamically changing the value, we either have to add our special binding syntax to show the property panel OR you'd need to setup a `TextBox` or `Slider` in the XAML example to bind the value too. Everything else gets obliterated whenever you make a XAML change otherwise.